### DISCUSSION TITLE
Fluent v8 LTS/EOL (long-term support / end of life)?

### DISCUSSION THREAD

**Discussion by mark-wiemer** [microsoft/fluentui#29100] [Q&A] [answered]
I know v9 is still relatively new, but it'd be great to have docs on how long v8 will receive security, a11y, or bugfix patches. Our team has a huge project to migrate and would like to start long-term planning, but we also want to wait until some more v9 components from the roadmap are delivered. Any updates here?

(The only similar discussion I see is from March 2022 and the asker closed it saying "it'll be around for a while", linking a page last updated September 2022.)

**Comment by mwiemer-microsoft** [comment]
Hi @micahgodbolt sorry to ping, but we are planning future work and that depends on whether there will be support for Fluent v8 through the next six months.

**Reply by blendsdk** [reply]
I am in a similar situation. I am planning to migrate a fairly large application to v9. At the moment the show stopper for us is the DataGrid's performance and several other issues. I don't want to use a third party DataGrid (even though some are extremely good). Because there is no information regarding the v8 EOL I am going to keep the production on v8. One thing for sure is that migrating from v8 to v9 is no small feat. For me it just means "File->New Project".  As an experiment I have been playing with an intermediate solution where  I create custom components resembling v9 API but wrapping v8 components. This way I hope the migration to v9 would be easier in 6-12 months time.

**Reply by sydsutton** [reply]
@mark-wiemer @blendsdk I saw that there had only been 1 Microsoft employee to reply to any discussion within the last 30 days, so I decided to reach out to @layershifter directly on X. His response: "For v8 - it's hard to say, it will stay around for some time. For v0 - already gets only fixes and we expect to deprecate it next year." . I hope this helps even a little bit.

**Reply by blendsdk** [reply]
Thank you. I will have to stay with v8 for as long as possible and rewrite my code to v9 slowly.

**Reply by mark-wiemer** [reply]
Thanks @sydsutton. For the record, I'm also an MSFT employee but obviously not with the Fluent team. I'll keep this issue updated if I learn more :)

**Reply by ling1726** [reply]
@blendsdk  #29908 has been resolve - are there any other issues?

**Reply by mwiemer-microsoft** [reply]
I haven't bumped into any answers, I'll search around and keep this discussion updated :)

**Reply by micahgodbolt** [reply, marked as answer]
As of right now, we don't have a specific EOL date in mind for v8. We're actively working with a number of teams to help them migrate from v8 to v9, and once that work is complete, we'll evaluate how long we can continue to support the older version.
